cypress/utils/profil.js

```diff
@@ -9,9 +9,8 @@ const fill_first_name = (prenom) => {
 }
 
 const fill_date_naissance = (birthDate) => {
-  cy.url().should("include", "date_naissance")
-  cy.get("#date_naissance").type(birthDate)
-  submit()
+  trySubmitWithoutAnswer("date_naissance")
+  fillDate("date_naissance", "#date_naissance", birthDate)
```

Comment:
Je pense qu'externaliser ce test rend les choses plus complexes. De plus avec l'implémentation actuelle, cette fonction ne teste pas de façon autonome, le non-passage à l'écran suivant.



```suggestion
  cy.url().should("include", url)
  submit()
  cy.url().should("include", url)
  fillDate("date_naissance", "#date_naissance", birthDate)
```